modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -12,28 +12,36 @@ namespace access to the {CNVProductName} Operators.
 
 .Prerequisites
 
-* `OperatorGroup` and `CatalogSource` Custom Resource Definition objects (CRDs),
-both created in the `openshift-cnv` namespace
+* Create a project called `openshift-cnv`.
 
 .Procedure
 
 . Open a browser window and navigate to the {product-title} web console.
 
 . Navigate in the web console to the *Operators → OperatorHub* page.
 
-. Scroll or type a keyword into the *Filter by keyword* box to find the KubeVirt
-HyperConverged Cluster Operator, then click on it.
+. Scroll or type a keyword into the *Filter by keyword* box to find the
+{CNVProductNameStart} Operator, then click on it.
```

Comment:
why a capital case for CNVProductName here? It's not a beginning of the sentence. For example, a paragraph above, before "Prerequisites" has it with low case + Operator.

Reply:
Ah, good catch! I think I misinterpreted this guideline: https://github.com/ousleyp/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization

I missed the "defer to the product's capitalization style" part (not to mention that I wasn't consistent about this). I will change it back to lower case.